### PR TITLE
style-guide: add section on serial comma

### PR DESCRIPTION
**Pull request by SethFalco**
Added notes on the serial comma as discussed on Matrix/Gitter:
* https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$pJum15wC15-d0DoaVlT4njfG9-KUJwGgbB1OENxn46g?via=gitter.im&via=matrix.org&via=matrix.coredump.ch
* https://gitter.im/tldr-pages/tldr?at=60f5506182dd9050f5fc55e3

The example used is up for debate.
It was just the best thing I could think of, that might be relevant to the repository.

If anyone can come up with a better example, feel free to suggest it!
Alternatively, we could just take something from Wikipedia. I just thought it'd be better if we could create our own that we might reasonably come across in the repository.

### CODE REVIEW

File: contributing-guides/style-guide.md [hunk 1]
```diff
@@ -76,3 +76,18 @@ Use backticks on the following:
 1. Paths, ex. `package.json`, `/etc/package.json`.
 2. Extensions, ex. `.dll`.
 3. Commands, ex. `ls`.
+
+## Serial Comma
+When declaring a list of items with 3 or more elements, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma), also known as the Oxford comma.
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:81]: its also known as the Harvard Comma.

Comment (CleanMachine1) [reply contributing-guides/style-guide.md:81]: `elements` works better but what do you think about `items`. I have no strong opinion on `items` so I don't care.

**Review by einverne — APPROVED** [review]
It looks good.

**Review by marchersimon — APPROVED** [review]
Thanks

**Review by CleanMachine1 — APPROVED** [review]

File: contributing-guides/style-guide.md (hunk 1 above)
Comment (SethFalco) [reply contributing-guides/style-guide.md:81]: Any opinions on if we should mention Oxford/Harvard comma?

```suggestion
When declaring a list of items with 3 or more elements, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma), also known as the Oxford comma or Harvard comma.
```

In general, it could be better to reword this too, I kinda hate it. 🤔 

```suggestion
When declaring a list of 3 or more items, use a [serial comma](https://en.wikipedia.org/wiki/Serial_comma). (also known as an Oxford comma or Harvard comma)
```

* a list of items with 3 or more elements -> a list of 3 or more items
* change how we say "also known as"

Comment (CleanMachine1) [reply contributing-guides/style-guide.md:81]: I agree with your 2nd change however thinking about it, I think the `also known as the Oxford comma` is pointless as anyone that clicks on the wiki link will find that information anyway

Comment (SethFalco) [reply contributing-guides/style-guide.md:81]: The reason I think the Oxford comma part might be relevant is because back in school (and informal conversation) it's primarily referred to as the Oxford comma.

My partner also only knew it as an "Oxford comma" until now.

I have no problem removing it if others agree, this comment isn't indicating a preference. Only justifying why I put it there in the first place.

Comment (CleanMachine1) [reply contributing-guides/style-guide.md]: Might that just be a British name, since Oxford University.........

Comment (SethFalco) [reply contributing-guides/style-guide.md]: My partner is from Belgium. (and that's where I live)
I have no doubt the reason I call it "Oxford comma" is because I'm British, but most of my non-Brit friends also call it that.

**Review by sbrl — APPROVED** [review]
Looks good to me! Thanks, @SethFalco :-)

**Review by navarroaxel — APPROVED** [review]

File: contributing-guides/style-guide.md [hunk 2]
```diff
@@ -76,3 +76,18 @@ Use backticks on the following:
...
+
+> Delete the Git branches, tags and remotes.
+
+The example above does not use a serial comma, so this could mean one of two things:
+* Delete the Git branches named `tags` and `remotes`.
+* Delete all of the following, Git branches, Git tags, and Git remotes.
+
+This can be resolved by inserting a comma before the "and" or "or" in the final element in the list.
+
+> Delete Git branches, tags, and remotes.
```
Comment (SethFalco) [inline contributing-guides/style-guide.md:93]: _ahem_ Just a minor change to make it match the sentence above. (they were meant to be the same)

```suggestion
> Delete the Git branches, tags, and remotes.
```

File: contributing-guides/style-guide.md (hunk 1 above)
Comment (marchersimon) [reply contributing-guides/style-guide.md]: For a more objective perspective: <https://trends.google.com/trends/explore?geo=US&q=Oxford%20comma,Harvard%20comma,Serial%20comma>

Comment (SethFalco) [reply contributing-guides/style-guide.md]: @marchersimon Nice idea, I'll keep that in mind for future.
